src/wrapper/discord/messageEmbed.ts

```diff
@@ -0,0 +1,68 @@
+import { MessageEmbed } from "discord.js";
+
+interface Field {
+	key: string;
+	value: string;
+	inline?: boolean;
+}
+
+interface Embed {
+	color?: number|'random';
+	author?: string;
+	authorIcon?: string;
+	footer?: string;
+	footerIcon?: string;
+	image?: string;
+	description?: string;
+
+	fields?: Field[];
+}
+
+function randomColor() {
+	return "#000000".replace(/0/g, () => (~~(Math.random() * 16)).toString(16));
+}
+
+function safeFieldValue(input: string): string {
+	if (input == null || input == '') {
+		return "**Empty String**";
+	}
+	return input;
+}
+
+export default function createEmbed(input: Embed): MessageEmbed {
```

Comment:
Are there any kind of other embeds in the discord.js library? Should we call this createMessageEmbed instead perhaps?